ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/logic/l3leaf.j2

```diff
@@ -38,6 +38,8 @@
                                                l3leaf.defaults.spanning_tree_mode ) %}
 {%             set switch.spanning_tree_priority = l3leaf.node_groups[l3leaf_node_group].spanning_tree_priority | arista.avd.default(
                                                    l3leaf.defaults.spanning_tree_priority ) %}
+{%             set switch.spanning_tree_root_super = l3leaf.node_groups[l3leaf_node_group].spanning_tree_root_super | arista.avd.default(
```

Comment:
change default to `| arista.avd.default( l3leaf.defaults.spanning_tree_root_super )` so we support setting this on all leafs.